Readme.md

```diff
@@ -64,9 +64,12 @@ If you do not add credentials to your profile, a credential prompt displays when
     - **True**: Reject connections with self-signed certificates.
     - **False**: Accept connections with self-signed certificates.
     
-Your profile is now available in the panel on the left. You can also use this profile in Zowe CLI directly.
-
-If you have multiple profiles, you might wish to delete some once you no longer need them. To do so, click on the trash can icon on the same line as the profile name.
+    Your profile is now available in the panel on the left. You can also use this profile in Zowe CLI directly.
+    
+**Rename or Delete Profiles**   
+    
+- To change the profile name, click the pencil icon next to the profile you wish to change. Ensure that the new profile name is unique.
```

Comment:
@DStatWriter 
I can confirm that profiles cannot be renamed from the explorer. Also there is no pencil icon present to do so. You would have to correct these lines. Thanks

Reply:
zdmullen 2 days ago Member
I would just say that you can edit just the names of the data sources (configurations). We have not yet added the ability to edit profiles here.

How does the user edit the name of a configuration? Pencil icon next to the configuration name in the tree?

Reply:
Yes, that is correct. That icon remained in place and has the same purpose.

Reply:
OK, reworded and moved under the "working with configurations" section.

Reply:
Great thanks Zeb